.github/workflows/push.yaml

```diff
@@ -14,6 +14,13 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Run build-dev.
+        id: build_dev
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.SECRET_ACCESS_KEY }}
```

Comment:
Question (non-blocking): Do we need to login to do the push?

Reply:
Yep.